.github/workflows/manual-docker-build.yml

```
@@ -145,6 +152,17 @@ jobs:
     secrets:
       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+  build-snowbunk:
+    needs: prepare
+    if: needs.prepare.outputs.snowbunk == 'true'
+    uses: ./.github/workflows/docker-build-reusable.yml
+    with:
+      container_name: snowbunk
+      context_path: containers/starbunk-dnd
```

actionlint (1.7.7)

input "context_path" is not defined in "./.github/workflows/docker-build-reusable.yml" reusable workflow. defined inputs are "container_name", "image_tag", "push_image", "registry"

(workflow-call)